docs/cn/instructions/eventmesh-runtime-protocol.md

```diff
@@ -290,7 +290,7 @@ message RequestHeader {
     string protocolDesc = 12;
 }
 
-message EventMeshMessage {
+message SimpleMessage {
```

Comment:
What about using `MessageRequest`?

Reply:
in my opinion, the naming need to be consistent across the protocol.

If we choose to use MessageRequest, then we have to add "Request" to the other message names as well: "BatchMessageRequest", "SubscriptionRequest", "HeartBeatRequest".

But I also think adding the word "Request" is a little bit redundant, because this is already a GRPC request message in nature.

Reply:
Ok, I just worried that someone might mistakenly think this is a message type.